academy/xl-run-prod/3-genesis.md

```diff
@@ -0,0 +1,232 @@
+
+Each validator operator then has to run a `gentx` command. **This command is not to be run on the server**. Instead it is run on the computer that hosts the _cold_ validator operator app key, using the keyring of your choice. Collect the consensus public key from Tendermint KMS, for instance `{"@type":"/cosmos.crypto.ed25519.PubKey","key":"byefX/uKpgTsyrcAZKrmYYoFiXG0tmTOOaJFziO3D+E="}`.
+
+If this is Alice, she may run:
+
+```sh
+$ ./myprojectd gentx cavedweller1nw793j9xvdzl2uc9ly8fas5tcfwfetercpdfqq \
+    3000000000stone \
+    --account-number 0 --sequence 0 \
+    --chain-id stone-age \
+    --pubkey '{"@type":"/cosmos.crypto.ed25519.PubKey","key":"byefX/uKpgTsyrcAZKrmYYoFiXG0tmTOOaJFziO3D+E="}'
+```
+
+<!--
+TODO Detail which keyring to use
+-->
+
+This creates a JSON file on the validator's computer, typically in `~/.myprojectd/config/gentx/` with the following form:
+
+```json
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+        "description": {
+          "moniker": "test",
+          "identity": "",
+          "website": "",
+          "security_contact": "",
+          "details": ""
+        },
+        "commission": {
+          "rate": "0.100000000000000000",
+          "max_rate": "0.200000000000000000",
+          "max_change_rate": "0.010000000000000000"
+        },
+        "min_self_delegation": "1",
+        "delegator_address": "cavedweller15fsdc94dykrztg3rc70zu53ecsta23m0n0hhmr",
+        "validator_address": "cavedwellervaloper15fsdc94dykrztg3rc70zu53ecsta23m0kmrzhs",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "byefX/uKpgTsyrcAZKrmYYoFiXG0tmTOOaJFziO3D+E="
+        },
+        "value": {
+          "denom": "stake",
+          "amount": "1000000"
+        }
+      }
+    ],
+    "memo": "ce1c54ea7a2c50b4b9f2f869faf8fa4d1a1cf43a@192.168.100.5:26656",
+    "timeout_height": "0",
+    "extension_options": [],
+    "non_critical_extension_options": []
+  },
+  "auth_info": {
+    "signer_infos": [
+      {
+        "public_key": {
+          "@type": "/cosmos.crypto.secp256k1.PubKey",
+          "key": "AlQbw3YOqdKHQ/bud+27qi8dVOGVoT9XFUzFGsTz/0qn"
+        },
+        "mode_info": {
+          "single": {
+            "mode": "SIGN_MODE_DIRECT"
+          }
+        },
+        "sequence": "0"
+      }
+    ],
+    "fee": {
+      "amount": [],
+      "gas_limit": "200000",
+      "payer": "",
+      "granter": ""
+    },
+    "tip": null
+  },
+  "signatures": [
+    "MKrPz/5eBouD116vj20yvdaFrRJC74d3pDofAePYSNFbvYPZ5bQIO/QatqUQvbSFIuoej6ahePby1Yob7AiVyg=="
+  ]
+}
+```
+
+<!--
+
+TODO include better example of JSON file
+
+-->
+
+Because a validator can be configured with multiple flags, look at:
+
+```sh
+$ ./myprojectd gentx --help
+```
+
+Each validator then returns you their transaction file(s).
+
+https://blog.althea.net/making-a-cosmos-chain/
```

Comment:
@xavierlepretre please include for the URL some context, i.e. make it a proper sentence.